### ISSUE TITLE
dirt dir missing in npm package

### ISSUE THREAD

**Issue by derhuerst** [wireapp/wire-web-core#16]
When installing via npm, the `dist` dir is missing. Therefore, requiring `@wireapp/core` fails.

**Comment by bennyn** [comment]
@derhuerst Are you sure about that? I just verified an installation via `npm i -S @wireapp/core` and got the `dist` folder:

![Screenshot](https://user-images.githubusercontent.com/469989/32281707-ed87f22a-bf1f-11e7-9c44-b20e54359fe6.PNG)

We also do explicitly upload `dist/commonjs` to [npm](https://www.npmjs.com/): 
- https://github.com/wireapp/wire-web-core/blob/v0.0.3/.npmignore

**Comment by derhuerst** [comment]
Can't reproduce. https://asciinema.org/a/145513

**Comment by bennyn** [comment]
@derhuerst Thanks for reporting back. It's really strange that it doesn't work on your system. Are you maybe symlinking `@wireapp/core`? 

I just tested on [runkit](https://runkit.com/) and it's resolved just fine:
- https://runkit.com/bennyn/59fb0eec1c4e1e001307cc73

Can you try if the dependency gets resolved with `yarn`? Our CI system (Travis) is also using `@wireapp/core` and it works:
- https://github.com/wireapp/wire-webapp/pull/1868/files

Really want to know why it seems to be different on your machine.

**Comment by derhuerst** [comment]
With `yarn`, it works: https://asciinema.org/a/145525

**Comment by derhuerst** [comment]
The tarball contains everything that is necessary. (Although there's also unnecessary stuff. 😉)

```shell
wget $(npm info @wireapp/core dist.tarball)
open core-0.0.3.tgz
tree package
```

```
package
├── LICENSE
├── README.md
├── dist
│   └── commonjs
│       ├── Account.d.ts
│       ├── Account.js
│       ├── index.d.ts
│       ├── index.js
│       └── payload
│           ├── PayloadBundle.d.ts
│           ├── PayloadBundle.js
│           ├── SessionPayloadBundle.d.ts
│           ├── SessionPayloadBundle.js
│           ├── index.d.ts
│           └── index.js
├── package.json
├── src
│   ├── main
│   │   ├── Account.ts
│   │   ├── index.ts
│   │   └── payload
│   │       ├── PayloadBundle.ts
│   │       ├── SessionPayloadBundle.ts
│   │       └── index.ts
│   └── test
│       ├── browser
│       │   └── index.js
│       └── node
│           ├── spec
│           │   └── Account.test.js
│           └── support
│               └── jasmine.json
└── tsconfig.json

11 directories, 22 files
```

I assume it's related to the `.npmignore` file.

**Comment by bennyn** [comment]
@derhuerst We just released [v0.0.4](https://github.com/wireapp/wire-web-core/releases/tag/v0.0.4). It is much tidier:

```
package
├── dist
│   └── commonjs
│       ├── Account.d.ts
│       ├── Account.js
│       ├── index.d.ts
│       ├── index.js
│       └── payload
│           ├── index.d.ts
│           ├── index.js
│           ├── PayloadBundle.d.ts
│           ├── PayloadBundle.js
│           ├── SessionPayloadBundle.d.ts
│           └── SessionPayloadBundle.js
├── LICENSE
├── package.json
└── README.md
```

Can you test if you can install v0.0.4 via npm?

**Comment by derhuerst** [comment]
It works now, thanks!
